scalastyle-config.xml

```diff
@@ -113,7 +113,6 @@
    <parameter name="maxMethods"><![CDATA[30]]></parameter>
   </parameters>
  </check>
- <check level="error" class="org.scalastyle.scalariform.PublicMethodsHaveTypeChecker" enabled="true"></check>
```

Comment:
This was complaining when writing the JdbcTestFixtures with Coffees class defined in object root, which makes it public. I don't want to define types there as it would become less clear and harder to read. If style checker complains from things I feel is wrong, I thought that removing it would be better. Might be that there are other places it adds more value than takes?

Reply:
Actually I added it back and didn't see the problem. Never mind, was something temporary I probably misunderstood. Added back.